book/src/user/target-tier-policies.md

```diff
@@ -0,0 +1,288 @@
+  block on the PR based on a tier 3 target. Do not send automated messages or
+  notifications (via any medium, including via `@`) to a PR author or others
+  involved with a PR regarding a tier 3 target, unless they have opted into
+  such messages.
+- Patches adding or updating tier 3 targets must not break any existing tier 2
+  or tier 1 target, and must not knowingly break another tier 3 target without
+  approval of either the core engineering team or the maintainers of the other
+  tier 3 target.
+
+If a tier 3 target stops meeting these requirements, or the target maintainers
+no longer have interest or time, or the target shows no signs of activity and
+has not built for some time, or removing the target would improve the quality
+of the Zebra codebase, we may post a PR to remove it; any such PR will be CCed
+to the target maintainers (and potentially other people who have previously
+worked on the target), to check potential interest in improving the situation.
+
+## Tier 2 target policy
+
+At this tier, the Zebra project guarantees that a target builds, and will reject
+patches that fail to build on a target. Thus, we place requirements that ensure
+the target will not block forward progress of the Zebra project.
+
+A proposed new tier 2 target must be reviewed and approved by the core
+engineering team based on these requirements.
+
+In addition, the devops team must approve the integration of the target
+into Continuous Integration (CI), and the tier 2 CI-related requirements. This
+review and approval may take place in a PR adding the target to CI, or simply
+by a devops team member reporting the outcome of a team discussion.
+
+- A tier 2 target must have value to people other than its maintainers. (It may
+  still be a niche target, but it must not be exclusively useful for an
+  inherently closed group.)
+- A tier 2 target must have a designated team of developers (the "target
+  maintainers") available to consult on target-specific build-breaking issues,
+  or if necessary to develop target-specific language or library implementation
+  details. This team must have at least 1 developer.
+- The target must not place undue burden on Zebra developers not specifically
+  concerned with that target. Zebra developers are expected to not gratuitously
+  break a tier 2 target, but are not expected to become experts in every tier 2
+  target, and are not expected to provide target-specific implementations for
+  every tier 2 target.
+- The target must provide documentation for the Zebra community explaining how
+  to build for the target using cross-compilation, and explaining how to run
+  tests for the target. If at all possible, this documentation should show how
+  to run Zebra programs and tests for the target using emulation, to allow
+  anyone to do so. If the target cannot be feasibly emulated, the documentation
+  should explain how to obtain and work with physical hardware, cloud systems,
+  or equivalent.
```

Comment:
All the emulation and cross-compilation requirements don't make sense for Zebra, please delete them in the whole document.

Reply:
Docker in an M1 is being emulated for some platforms. This could apply if we consider this scenario.

Reply:
I'm leaving some parts which refers to "emulation" as we've hit some of the constraints ourself. For example, we wanted to build ARM64 to include support for M1 devices, but the emulation wasn't good enough and took way too much time.

As stated in the Tier 1 platform policy, this applies to us:

>     In particular, if building the platform takes a reasonable amount of time,
>     but the platform cannot run the testsuite in a timely fashion due to low
>     performance of either native code or accurate emulation, that alone may
>     prevent the platform from qualifying as tier 1.


Other statements in this policy start with `If`, so it's not exactly stating a "present" state, it might be a future requirement. The policy is just taking in into consideration so we don't have to change it if new requirements are introduced.